### PR TITLE
Added:- "View all Packages from Author"

### PR DESCRIPTION
**Pull request by iammuhammadahmad**
Hi,
I added the functionality "View all Packages from the author". 

Please verify and let me know if anything is changeable.
Thanks

### CODE REVIEW

File: lib/all.js [hunk 1]
```diff
@@ -1,13 +1,12 @@
 module.exports = function (router) {
-  router.get('/all.json', async function (req, res) {
```
Comment (twodayslate) [inline lib/all.js:2]: Why is this being removed?  I think this was supposed to be copied instead of moved.

File: models/package.js [hunk 2]
```diff
@@ -39,6 +39,8 @@ module.exports = (sequelize, DataTypes) => {
       // This way, the current date/time will be used to populate this column (at the moment of insertion)
     }
   }, {})
+
+
```
Comment (twodayslate) [inline models/package.js:43]:
```suggestion
```

File: lib/oauth.js [hunk 3]
```diff
@@ -63,6 +63,7 @@ module.exports = function (router) {
       client_id: process.env.GITHUB_CLIENT_ID,
       redirect_uri: `${protocol}://${host}/auth/github/callback`
     })
+    console.log("client herereerere"+params)
```
Comment (twodayslate) [inline lib/oauth.js:66]:
```suggestion
```

File: lib/oauth.js [hunk 4]
```diff
@@ -100,6 +101,7 @@ module.exports = function (router) {
       return
     }
     res.render('login')
+
```
Comment (twodayslate) [inline lib/oauth.js:104]:
```suggestion
```

File: index.js [hunk 5]
```diff
@@ -78,30 +78,28 @@ module.exports = async function (app) {
...
     const octokit = await app.auth()
     const { data } = await octokit.apps.getAuthenticated()
     res.json(data)
   })
 
+
+
+
+
+
```
Comment (twodayslate) [inline index.js:102]:
```suggestion
```

File: index.js [hunk 6]
```diff
@@ -78,30 +78,28 @@ module.exports = async function (app) {
   require('./lib/index')(expressApp)
   require('./lib/about')(expressApp)
   require('./lib/add')(expressApp)
+  require('./lib/all')(expressApp)
   require('./lib/package')(expressApp)
   require('./lib/search')(expressApp)
   require('./lib/mod')(expressApp)
   require('./lib/delete')(expressApp)
-  require('./lib/api/all')(expressApp)
```
Comment (twodayslate) [inline index.js:85]: why is an API endpoint being removed?

**Review by twodayslate — CHANGES_REQUESTED** [review]

File: lib/all.js (hunk 1 above)
Comment (iammuhammadahmad) [reply lib/all.js:2]: Oh Sorry, Yes you are right. Just give me few minuts. I add it

File: action/index.js [hunk 7]
```diff
@@ -1,95 +1,22 @@
-module.exports =
```
Comment (twodayslate) [inline action/index.js:1]: Why was this changed?
